src/main/java/app/gpx_animator/core/data/MusicCodec.java

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2023 Contributors to the Sports-club
```

Comment:
Please change this with the correct project name. We usually have the following line here: `Copyright Contributors to the GPX Animator project.`

Reply:
changed copyright